libraries/WebServer/src/WebServer.cpp

```diff
@@ -796,3 +798,14 @@ String WebServer::_responseCodeToString(int code) {
     default:  return F("");
   }
 }
+
+bool ON_STA_FILTER(WebServer &server) {
+  return (
+    (WiFi.localIP() != IPAddress(0, 0, 0, 0) && server.client().localIP() != IPAddress(0, 0, 0, 0))
+    && WiFi.localIP() == server.client().localIP()
+  );
+}
+
+bool ON_AP_FILTER(WebServer &server) {
+  return WiFi.localIP() == server.client().localIP();
```

Comment:
This will check STA again. See above suggestion

Reply:
Done